mmv1/templates/terraform/examples/firebasedataconnect_service_basic.tf.tmpl

```diff
@@ -0,0 +1,26 @@
+# Enable Firebase Data Connect API
+resource "google_project_service" "fdc" {
+  project = "{{index $.TestEnvVars "project_id"}}"
+  service = "firebasedataconnect.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Create an FDC service
+resource "google_firebase_data_connect_service" "default" {
+  project = "{{index $.TestEnvVars "project_id"}}"
+  location = "us-central1"
+  service_id = "{{index $.Vars "service_id"}}"
+  # TODO(b/394642094): Cover force deletion once it's supported
```

Comment:
This will show up in public documentation. You definitely don't want a buganizer number there.

Why is force deletion not supported? It's supported in the API.

Reply:
Force deletion isn't supported on our API yet either. It's in the process of getting rolled out but will take several weeks before it goes out fully in Prod